src/magic_folder/cli.py

```diff
@@ -141,6 +146,12 @@ class InitializeOptions(usage.Options):
          "(Optional) the Twisted client-string for our REST API (only required "
          "if auto-converting from the --listen-endpoint fails)"),
     ]
+    optFlags = [
+        ("websocket-status", "s",
+         "Provide the WebSocket status endpoint. If this is enabled all status"
+         " updates are saved when no clients are connected."),
+    ]
```

Comment:
The main reason I made it optional is because if nothing ever connects to it, we'll consume unbounded memory. The other solution there would be some option to turn that specific behavior on or off...

Reply:
I'm not sure that it makes sense to keep track of messages when nobody is connected (particularly not an unbounded amount). Given that we don't persist a history, I think any client will need to get the current state independent of the websocket.

My impression of how anything subscribing to a status feed like this works is 1) connect to the status feed, 2) query current state using info from the feed to incrementally update it.

Reply:
as per https://github.com/LeastAuthority/magic-folder/issues/335#issuecomment-847508142 I re-shaped the way the API works for clients.